src/apis/MessagesEndpoint.es6.js

```diff
@@ -11,7 +11,16 @@ const TYPE_MAP = {
 
 const parseGetBody = (res, apiResponse, apiRequest) => {
   const { body } = res;
-  body.data.children.forEach(c => apiResponse.addResult(TYPE_MAP[c.kind].fromJSON(c.data)));
+  body.data.children.forEach(c => {
+    const replies = [];
+    c.data.replies && c.data.replies.data.children.forEach(r => {
```

Comment:
Oh, hah!  I had it both ways and went with this but I can put it in an if statement if it makes you more comfortable.